apiary.apib

```diff
@@ -2903,6 +2903,13 @@ Get a list of project milestones.
                     + Members
                         + open
                         + closed
+                + billing_method (object)
+                    + One Of
+                        + Properties
+                            + method: `time_and_materials` (string)
+                        + Properties
+                            + method: `fixed_price` (string)
+                            + price (Money)
```

Comment:
B: We should remove the price here. The price will be exposed under the "profit" information.